lib/code_climate/test_reporter.rb

```diff
@@ -20,8 +20,8 @@ def self.environment_variable_set?
       return @environment_variable_set if defined?(@environment_variable_set)
 
       @environment_variable_set = !!ENV["CODECLIMATE_REPO_TOKEN"]
-      unless @environment_variable_set
-        logger.info("Not reporting to Code Climate because ENV['CODECLIMATE_REPO_TOKEN'] is not set.")
+      if @environment_variable_set
+        logger.debug("Reporting coverage data to Code Climate.")
```

Comment:
It's helpful to print this on info level, so that we can see when customers write in with issues if the environment variable was set.